libraries/AP_Scripting/generator/description/bindings.desc

```diff
@@ -691,13 +691,16 @@ singleton AP_EFI method get_state void EFI_State'Ref
 
 -- ----END EFI Library----
 
+include AP_Logger/AP_Logger.h
 singleton AP_Logger rename logger
 singleton AP_Logger manual write AP_Logger_Write 7
+singleton AP_Logger method log_file_content void string
 
 singleton i2c manual get_device lua_get_i2c_device 4
 
 global manual millis lua_millis 0
 global manual micros lua_micros 0
+global manual utc_micros lua_utc_micros 0
```

Comment:
I suggest we wait and see if we get double first. 

In anycase I don't think its worth adding a binding just to log, the log review tool should have all the data it needs to interpolate.

If we decide we want the binding anyway why not add `AP::rtc` as a singleton so we can add more rtc methods in the future? `get_local_time` is one that I can see might be useful in the future.

Reply:
i've removed this binding